cmd/sidecars/network-passt-binding/domain/configurator.go

```diff
@@ -84,6 +84,16 @@ func NewPasstNetworkConfigurator(ifaces []vmschema.Interface, networks []vmschem
 }
 
 func (p PasstNetworkConfigurator) Mutate(domainSpec *domainschema.DomainSpec) (*domainschema.DomainSpec, error) {
+	const (
+		sharedMemoryBackingAccessMode = "shared"
+		memfdMemoryBackingSourceType  = "memfd"
+	)
+
+	if domainSpec.MemoryBacking != nil && domainSpec.MemoryBacking.Access != nil && domainSpec.MemoryBacking.Access.Mode != sharedMemoryBackingAccessMode {
+		return nil, fmt.Errorf("memory backing access mode must be 'shared'; cannot override existing mode: %s",
+			*domainSpec.MemoryBacking.Access)
```

Comment:
`domainSpec.MemoryBacking.Access.Mode` will be more explicit please and won't print the struct
i.e `Mode: {value}`
(since you already wrote `mode:` it is redundant)